src/Microsoft.AspNet.OData.Shared/Interfaces/IWebApiRequestMessage.cs

```diff
@@ -84,6 +84,15 @@ internal interface IWebApiRequestMessage
         /// Get the next page link for a given page size.
         /// </summary>
         /// <param name="pageSize">The page size.</param>
+        /// <param name="instance">The instance based on which the skiptoken value is generated</param>
+        /// <param name="objToSkipTokenValue">Function that takes in the last object and returns the skiptoken value string.</param>
+        /// <returns></returns>
+        Uri GetNextPageLink(int pageSize, object instance, Func<object, string> objToSkipTokenValue);
+
+        /// <summary>
+        /// Get the next page link for a given page size.
+        /// </summary>
+        /// <param name="pageSize">The page size</param>
         /// <returns></returns>
         Uri GetNextPageLink(int pageSize);
```

Comment:
Is this overload still required? It doesn't seem to be called from anywhere (at least in classic) and interface is internal; can we delete it?